### ISSUE TITLE
Remove the redundant "Start A New Submission" link when multiple submission dashboards are displayed

### ISSUE THREAD

**Issue by blesildaramirez** [pkp/pkp-lib#11074]
## Issue Description
When multiple submission dashboards are displayed (e.g., 'Dashboards' and 'My Submissions'), both menu sublinks include the 'Start a New Submission' button. Remove the button from the menu with the least permissible access.

## How to reproduce / Current behavior
OJS:
Login as a user with reviewer, author, and journal editor roles. Multiple submission dashboards should appear on the sidemenu.

OMP:
Login as a user with Author, Press manager, Internal Reviewer roles. Multiple submission dashboards should appear on the sidemenu.

## What application are you using?
 OJS, OMP 3.5.0

## Testing method
- Method: Manual

## Action
There should be only one "Start A New Submission" button. Remove the button from the menu with the least permissible access.

---
Pull requests:
_**main**_
pkp-lib: https://github.com/pkp/pkp-lib/pull/11077
ojs: https://github.com/pkp/ojs/pull/4737

_**stable-3_5_0**_
pkp-lib: https://github.com/pkp/pkp-lib/pull/11076
ojs: https://github.com/pkp/ojs/pull/4736
omp: https://github.com/pkp/omp/pull/1895
ops: https://github.com/pkp/ops/pull/912
